tests/db-setup.test.ts

```diff
@@ -305,11 +305,15 @@ describe('Recording a ReBench execution from payload files', () => {
 
       await db.awaitQuiescentTimelineUpdater();
 
-      expect(recMs).toEqual(459928);
+      const measurements2 = await db.query(
+        'SELECT SUM(cardinality(value)) FROM Measurement'
+      );
+      expect(recMs).toEqual(460);
       expect(recPs).toEqual(0);
-      expect(parseInt(measurements.rows[0].cnt)).toEqual(459928 + 4);
+      expect(parseInt(measurements.rows[0].cnt)).toEqual(460 + 4);
```

Comment:
Please don't follow my bad example. Please introduce named constants instead of relying on magic numbers.